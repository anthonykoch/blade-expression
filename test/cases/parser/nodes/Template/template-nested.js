`hello${`start${'lul'}end`}hey`
