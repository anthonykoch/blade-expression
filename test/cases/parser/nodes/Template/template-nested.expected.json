---
name: Template - nested
description: nested templates
---
[
  {
    "type": "ExpressionStatement",
    "expression": {
      "type": "TemplateLiteral",
      "expressions": [
        {
          "type": "TemplateLiteral",
          "expressions": [
            {
              "type": "Literal",
              "raw": "'lul'",
              "value": "lul",
              "start": 16,
              "end": 21
            }
          ],
          "quasis": [
            {
              "type": "TemplateElement",
              "value": {
                "raw": "start"
              },
              "tail": false,
              "start": 8,
              "end": 16
            },
            {
              "type": "TemplateElement",
              "value": {
                "raw": "end"
              },
              "tail": true,
              "start": 21,
              "end": 26
            }
          ],
          "start": 8,
          "end": 26
        }
      ],
      "quasis": [
        {
          "type": "TemplateElement",
          "value": {
            "raw": "hello"
          },
          "tail": false,
          "start": 0,
          "end": 8
        },
        {
          "type": "TemplateElement",
          "value": {
            "raw": "hey"
          },
          "tail": true,
          "start": 26,
          "end": 31
        }
      ],
      "start": 0,
      "end": 31
    },
    "start": 0,
    "end": 31
  }
]
